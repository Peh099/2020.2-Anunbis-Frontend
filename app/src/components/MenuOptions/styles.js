import styled from 'styled-components';

export const Container = styled.div`
  display: flex;
  flex-direction: column;
  background-color: inherit;
  border-bottom-left-radius: 10px;
  border-bottom-right-radius: 10px;
  z-index: 2;
  width: min(15vw, 130px);

  Button {
    font-size: 13px;
    margin-bottom: 15px;
    box-shadow: none;
    word-wrap: break-word;
    span:first-child {
      color: red;
    }
    @media (max-width: 550px) {
      font-size: 10px;
      margin-bottom: 13px;
    }
    @media (max-width: 460px) {
      font-size: 8px;
      margin-bottom: 9px;
    }
    @media (max-width: 300px) {
      font-size: 0px;
    }
    @media (max-height: 300px) {
      display: none;
    }
  }
`;
