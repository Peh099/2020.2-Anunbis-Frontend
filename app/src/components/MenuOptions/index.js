import React from 'react';
import { Container } from './styles.js';

export default function MenuOptions({ children }) {
  return (
    <Container>
      {children}
    </Container>
  );
}
