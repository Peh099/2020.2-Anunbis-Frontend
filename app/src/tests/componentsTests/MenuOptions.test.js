import renderer from 'react-test-renderer';
import MenuOptions from '../../components/MenuOptions';

describe('Snapshot MenuOption component', () => {
    it('matches the snapshot', () => {
        const tree = renderer.create(<MenuOptions />).toJSON()
    
    expect(tree).toMatchSnapshot()
    })
})