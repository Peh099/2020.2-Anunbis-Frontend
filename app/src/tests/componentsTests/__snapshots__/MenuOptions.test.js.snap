// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot MenuOption component matches the snapshot 1`] = `
<div
  className="sc-bdnxRM fypzih"
/>
`;
